tablesorter-grouping.js

```diff
@@ -45,8 +49,8 @@
       });
     });
 
-    // Clear the grouped rows
-    $("tr", table).removeClass(groupedRowClass);
+    // Reset all the classes
+    $("tr", table).removeClass();
```

Comment:
You can't remove all the classes from all the rows. The user may have added their own classes into the row, which you just destroyed.